examples/nwc/wallet-service/notifications.ts

```
@@ -0,0 +1,108 @@
+    });
+
+    return {
+      result: { preimage, fees_paid: feesPaid },
+      error: undefined,
+    };
+  },
+});
+
+console.info("Waiting for events...");
+process.on("SIGINT", function () {
+  console.info("Caught interrupt signal");
```

ESLint

[error] Unexpected console statement.

(no-console)